projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/README.md

```diff
@@ -0,0 +1,15 @@
+# WPCom Endpoints
+
+These are endpoints designed to be shared across both WPCom platforms: Simple and Atomic.
+
+These calls can be bridged from `public-api.wordpress.com` to an Atomic site seamlessly with a little extra work to ensure that the freshest data is always served. See PCYsg-liz-p2 for more.
+
+This is intended to replace:
```

Comment:
I'm not sure "replace" is the right word. The directories in the Jetpack plugin should still be used for things that are also needed on dotorg sites using Jetpack. This is for things that are needed on Simple and Atomic but _not_ dotorg sites.

Reply:
Thanks for the contextual clue, I was writing this from a wpcom-centric perspective, I'll make this more clear